app/services/subscriptions/organization_billing_service.rb

```diff
@@ -22,11 +22,15 @@ def call
           end
         end
 
-        BillSubscriptionJob.perform_later(
-          billing_subscriptions,
-          today.to_i,
-          invoicing_reason: :subscription_periodic
-        )
+        subscription_groups = group_by_payment_method(billing_subscriptions)
```

Comment:
For current subscriptions in the DB, without any migration, this method should return only one group and there should be no change in the logic.
@mariohd Can you double check the logic? Do you think we should still add additional feature flag guard?